src/Pygmentize/Pygmentize.php

```diff
@@ -50,7 +50,7 @@ public static function highlight($source, $language, $encoding = "utf-8", $forma
       else
         $args = sprintf(" -f %s -g -O encoding=%s,style=%s,lineos=1", $formatter, $encoding, $style);
 
-      $proc = proc_open(self::PIGMENTS_BINARY.$args, $dspec, $pipes);
+      $proc = proc_open(self::PIGMENTS_BINARY.escapeshellcmd($args), $dspec, $pipes);
```

Comment:
Input should be escaped by `escapeshellarg` instead and that should be applied to each argument separately.

See:

> http://php.net/manual/en/function.escapeshellcmd.php 
> Warning
> escapeshellcmd() should be used on the whole command string, and it still allows the attacker to pass arbitrary 
> number of arguments. For escaping a single argument escapeshellarg() should be used instead.

and http://php.net/manual/en/function.escapeshellarg.php